test/System.ServiceModel.Federation.Tests/WSTrustChannelSecurityTokenProviderTests.cs

```diff
@@ -373,6 +374,7 @@ public static TheoryData<ProofTokenGenerationTheoryData> ProofTokenTheoryData
                 {
                     // Computed key, key size from non-default SecurityAlgorithmSuite
                     RequestSecurityAlgorithmSuite = SecurityAlgorithmSuite.TripleDes,
+                    RequestKeySize = 0,
```

Comment:
y

Reply:
This is just fixing the test to not include a KeySize parameter in the RST. This is necessary so that we can test that the key size is picked based on the security algorithm suite. If a key size were preset in the RST, that would be used without ever looking at the security algorithm suite.

Reply:
merge